lib/Doctrine/ORM/Mapping/Driver/AnnotationDriver.php

```diff
@@ -29,8 +32,19 @@
 /**
  * The AnnotationDriver reads the mapping metadata from docblock annotations.
  */
-class AnnotationDriver extends AbstractAnnotationDriver
+class AnnotationDriver implements MappingDriver
```

Comment:
This BC break needs to be reverted, and moved to the next major.

Ref: https://github.com/doctrine/orm/issues/9668